tools/topology/topology1/sof-tgl-max98357a-rt5682-rtnr-16kHz.m4

```diff
@@ -0,0 +1,225 @@
+#
```

Comment:
Couldn't this be merged to the existing sof-tgl-max98357a-rt5682.m4?

Reply:
I tried running the topology sof-tgl-max98357a-rt5682 without modifications and did 16kHz recording using arecord. There is a kpb error shows that host_buffer_size (32000) must be at least 384000. Is there a setting to increase host buffer size?

Reply:
A flag could be added to determine the correct size if needed.

Reply:
Added RTNR_BUFFER_SIZE_MIN and RTNR_BUFFER_SIZE_MAX in pipe-rtnr-capture-16khz.m4. Thanks.